tests/accelerators/test_accelerator_connector.py

```diff
@@ -291,7 +291,7 @@ def test_accelererator_invalid_type_devices(mock_is_available, mock_device_count
         _ = Trainer(accelerator="gpu", devices=device_count)
 
 
-@RunIf(min_gpus=1)
+@RunIf(min_cuda_gpus=1)
```

Comment:
maybe make this rename as separate PR so it minimizes this change?

Reply:
Can do, but to me it is strongly coupled to this PR. Let me first finish here and then I can look into splitting :)